src/lib.rs

```diff
@@ -597,6 +604,7 @@ impl<K: Eq + Hash, V: Eq> NonBlockingHashMap<K, V> {
         }
     }
 
+    #[allow(dead_code)]
```

Comment:
Let's just keep the clippy warnings for these two functions (`fast_keyeq` and `print_table`) as I planned to revisit them later.